ng2-components/ng2-alfresco-core/src/services/AlfrescoAuthentication.service.ts

```diff
@@ -20,16 +20,14 @@ import { LogService } from './log.service';
 import { SettingsService } from './settings.service';
 import { StorageService } from './storage.service';
 import { AlfrescoApiService } from './alfresco-api.service';
-import { AuthService } from './auth.service';
 
 /** @deprecated AlfrescoAuthenticationService is deprecated. Use AuthService instead */
 @Injectable()
-export class AlfrescoAuthenticationService extends AuthService {
+export class AlfrescoAuthenticationService {
```

Comment:
If we keep it in this way it's going to be easy have double instance of the same class, with inspected behaviour, one for AlfrescoAuthenticationService and another for AuthService.